### DISCUSSION TITLE
What are we missing?

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#108] [Ideas]
Hey everyone, I would love to get an understanding of what we missed, please comment on a subject area and possibly some tooling that we have not covered so we can keep the content coming for this repository. 

I am already working on the next phase of the project but would love to hear your feedback.

**Comment by mbailey-eh** [comment]
Arguably AWS is a bigger and more common cloud provider compared to Azure. Will there be an AWS version?

**Reply by MichaelCade** [reply]
@mbailey-eh We are just kicking off the AWS content in the 2023 edition.

**Comment by MichaelCade** [comment]
I am going to say watch this space as we look at the next phase of the project. Overall AWS should be covered as should Google and others have been mentioned such as Oracle and RedHat.

**Comment by Itchimonji** [comment]
Hey Michael,
thanks for this great repository. It is very helpful in daily work.

For educational or local Kubernetes clusters I can recommend "Kind". It is like the same way as with Packer or with minikube, but a little bit more lightweight, easy to handle, and with any number of nodes. Maybe you can add it to your guide.

https://kind.sigs.k8s.io/

Thanks for all and keep up the good work :)
